.travis.yml

```diff
@@ -18,7 +18,7 @@ script:
   - |
       echo "Generating service crates..." &&
       ( cd service_crategen && \
-        cargo run -- generate -c ./services.json -o ../rusoto/services && \
```

Comment:
Curious why this `--no-format` is needed? Don't we want it to be formatted?

Reply:
I had this in here to try and make the builds not run into the timeout (which was largely successful). If we want the CI to format though, I can remove it.